src/encoded/audit/experiment.py

```diff
@@ -764,15 +767,17 @@ def check_experiment_rna_seq_standards(value,
         ['RNA-seq of long RNAs (paired-end, stranded)',
          'RNA-seq of long RNAs (single-end, unstranded)',
          'Small RNA-seq single-end pipeline',
-         'RAMPAGE (paired-end, stranded)'])
+         'RAMPAGE (paired-end, stranded)',
+         'Long read RNA-seq pipeline'])
     if pipeline_title is False:
         return
 
     standards_links = {
         'RNA-seq of long RNAs (paired-end, stranded)': ' /data-standards/rna-seq/long-rnas/ ',
         'RNA-seq of long RNAs (single-end, unstranded)': ' /data-standards/rna-seq/long-rnas/ ',
         'Small RNA-seq single-end pipeline': ' /data-standards/rna-seq/small-rnas/ ',
-        'RAMPAGE (paired-end, stranded)': ' /data-standards/rampage/  '
+        'RAMPAGE (paired-end, stranded)': ' /data-standards/rampage/  ',
+        'Long read RNA-seq pipeline': ' /data-standards/long-read-rna-pipeline/ '
```

Comment:
Not a suggestion for changes. It's just me not knowing for sure. It seems the linked has been moved: https://www.encodeproject.org/data-standards/rampage/. Would it be better to use the actual link? Or the idea is to maintain the consistency which might be good for maintenance?

Reply:
I'm not sure if there is a ticket for this, but I think that at some point we will need to review these links, especially since the documentation structure was overhauled